components/booking/BookingListItem.tsx

```diff
@@ -109,7 +109,7 @@ function BookingListItem(booking: BookingItem) {
         </div>
         {booking.description && (
           <div className="text-sm text-gray-500 truncate max-w-52 md:max-w-96" title={booking.description}>
-            &quot;{booking.description}&quot;
+            &quot;{booking.description && booking.description.replace(/(<([^>]+)>)/gi, "")}&quot;
```

Comment:
Does this issue happen in only this place? It will be better to create a virtual field e.g. short-description instead of doing this in the frontend component. It will be more scalable.

Reply:
i agree. i also cant reproduce this so we may just close it